app.js

```diff
@@ -22,4 +22,22 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+  .getBeers()
+  .then(beers=> res.render('beers',{beer:beers}  ))
+  .catch(error => console.log(error));
+  
+});
+
+app.get('/random-beers', (req,res) => {
+  punkAPI
+  .getRandom()
+  .then(randomBeer=> {
+    res.render('random-beer', {randomBeer:randomBeer})
+    console.log(randomBeer)
+  })
+  .catch(error => console.log(error))
+})
+
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
```

Comment:
Great job Mahmut! Keep it up! When you have some time you can go over the bonuses. I'm sure they will be no brainer for you ;)